stream-chat-android-ui-components-sample/src/main/AndroidManifest.xml

```diff
@@ -25,5 +26,11 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+
+        <activity
```

Comment:
I think we shouldn't override default styles in our sample app. This will allow catching missing default styles  earlier